<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开启激光器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>连接串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>断开串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>端口号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="com">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox QAbstractItemView::item { min-height: 50px; min-width: 60px; }</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="freshButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>功率：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设定</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="power">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>42</width>
      <height>22</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9400</number>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭激光器</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_seedOpen">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_seedSet">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>种子源</string>
    </property>
   </widget>
   <widget class="QPushButton" name="freshButton_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>180</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>430</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>功率：</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_seedOpen_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>310</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="state_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>550</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="power_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>42</width>
      <height>22</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9400</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_seedSet_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>440</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>放大器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>端口号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="com_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox QAbstractItemView::item { min-height: 50px; min-width: 60px; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开启激光器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>300</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭激光器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>180</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>断开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>连接串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>31</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uw</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>430</y>
      <width>31</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uw</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
